packages/insomnia-app/app/ui/components/modals/generate-config-modal.tsx

```diff
@@ -49,27 +49,25 @@ export class GenerateConfigModal extends PureComponent<{}, State> {
     this.modal = n;
   }
 
-  async _generate(generatePlugin: ConfigGenerator, apiSpec: ApiSpec) {
+  async _generate(generatePlugin: ConfigGenerator, apiSpec: ApiSpec): Promise<Config> {
```

Comment:
```suggestion
  async _generate(generatePlugin: ConfigGenerator, apiSpec: ApiSpec) {
```
The config is already typed on the next line, so the return type is secured.

Reply:
Thanks